internal/progress.go

```diff
@@ -73,6 +69,10 @@ func (p *Progress) MaybeReport(progress int64) {
 		}
 		if pct > p.pct {
 			p.pct = pct
+		}
+		if p.fractional {
```

Comment:
Why is the fractional and pct progress different ?

Reply:
It was a part of initial implementation and is used while showing progress for foreign key update and  data written in the cmd line respectively.